frontend-react/src/pages/details/FacilitiesTable.tsx

```diff
@@ -11,14 +9,11 @@ interface FacilitiesTableProps {
 
 function FacilitiesTable(props: FacilitiesTableProps) {
     const { reportId }: FacilitiesTableProps = props;
-    const facilities: FacilityResource[] = useResource(
```

Comment:
Will have a separate PR after this merges to remove the FacilityResource and any other code we can remove from previous integrations